src/toast/io/observation_hdf_load_v0.py

```diff
@@ -0,0 +1,132 @@
+# Copyright (c) 2021-2021 by the parties listed in the AUTHORS file.
```

Comment:
Should this file be included in the PR?

Reply:
Yes, the idea is to always support reading past versions of the file format, but only support writing the latest format.  As the format evolves, some of the old reading code will be moved out of the way into separate files like this.